openwrt-openwisp-monitoring/files/monitoring.init

```diff
@@ -50,10 +51,12 @@ start_service() {
     fi
     interval="--interval $interval"
     monitored_interfaces="--monitored_interfaces $monitored_interfaces"
```

Comment:
does 

```
monitored_interfaces="--monitored_interfaces \"$monitored_interfaces\""
```

allow us to not having to wrap the value in quotes in the UCI config file?